Failed to release assets for Windows at 0.13.3

https://github.com/alexpasmantier/television/releases/tag/0.13.3

https://github.com/alexpasmantier/television/actions/runs/16948670666/job/48036009551

```
error[E0433]: failed to resolve: could not find `unix` in `os`
Error:   --> television\utils\command.rs:13:9
   |
13 |     os::unix::process::CommandExt,
   |         ^^^^ could not find `unix` in `os`
   |
note: found an item that was configured out
  --> /rustc/29483883eed69d5fb4db01964cdf2af4d86e9cb2\library\std\src\os\mod.rs:36:9
note: the item is gated here
  --> /rustc/29483883eed69d5fb4db01964cdf2af4d86e9cb2\library\std\src\os\mod.rs:28:1
note: found an item that was configured out
  --> /rustc/29483883eed69d5fb4db01964cdf2af4d86e9cb2\library\std\src\os\mod.rs:85:9
note: the item is gated here
  --> /rustc/29483883eed69d5fb4db01964cdf2af4d86e9cb2\library\std\src\os\mod.rs:84:1

error[E0599]: no method named `exec` found for struct `std::process::Command` in the current scope
Error:    --> television\utils\command.rs:175:27
    |
175 |             let err = cmd.exec();
    |                           ^^^^ method not found in `std::process::Command`

Some errors have detailed explanations: E0433, E0599.
For more information about an error, try `rustc --explain E0433`.
error: could not compile `television` (lib) due to 2 previous errors
Error: The process 'C:\Users\runneradmin\.cargo\bin\cargo.exe' failed with exit code 101
```

--- a/television/utils/command.rs
+++ b/television/utils/command.rs
@@ -8,9 +8,10 @@ use crate::{
 use anyhow::Result;
 use lazy_regex::{Lazy, Regex, regex};
 use rustc_hash::FxHashSet;
+#[cfg(unix)]
+use std::os::unix::process::CommandExt;
 use std::{
     collections::HashMap,
-    os::unix::process::CommandExt,
     process::{Command, ExitStatus, Stdio},
 };
 use tracing::debug;
@@ -170,6 +171,7 @@ pub fn execute_action(
         &action_spec.command.env,
     );
 
+    #[cfg(unix)]
     match action_spec.mode {
         ExecutionMode::Execute => {
             let err = cmd.exec();
@@ -184,6 +186,22 @@ pub fn execute_action(
             let mut child = cmd.spawn()?;
             Ok(child.wait()?)
         }
+    }
+
+    // On windows we can't replace the current process, so we always fork
+    #[cfg(not(unix))]
+    {
+        if action_spec.mode == ExecutionMode::Execute {
+            debug!(
+                "ExecutionMode::Execute is not supported on Windows. Falling back to Fork."
+            );
+        }
+        cmd.stdin(Stdio::inherit())
+            .stdout(Stdio::inherit())
+            .stderr(Stdio::inherit());
+
+        let mut child = cmd.spawn()?;
+        Ok(child.wait()?)
     }
 }
 

